rest/rest_test.go

```diff
@@ -207,7 +243,69 @@ func TestCORS(t *testing.T) {
 			for key, value := range tt.headers {
 				assert.Equal(t, value, resp.Header.Get(key))
 			}
+
+			if tt.wantResponse != nil {
+				tt.wantResponse(t, resp)
+			}
 		})
 	}
+}
 
+func Test_corsConfig_OriginAllowed(t *testing.T) {
```

Comment:
I think I use the form "TestCorsConfig_OriginAllowed". But your way is probably the cleanest one since it is more readable and allows differentiating between exported and unexported objects.

Reply:
This was the way it was autogenerated 🤷

Reply:
Right, me too. But when I autogenerate, e.g., issueToken I get `TestService_issueToken`

Reply:
The "logic" seems to be Test*MyType*_*myFunction* for exported types and Test_*myType*_*myFunction* for unexpected types.